workspace/api/openfdd_bridge/routes/model_routes.py

```
@@ -26,13 +31,50 @@ def _ttl() -> TtlService:
+        for site in sites:
+            if isinstance(site, dict) and str(site.get("id")) == sid:
+                site["name"] = body.name.strip()
+                break
+        else:
+            sites.insert(0, {"id": sid, "name": body.name.strip()})
+    path = _ttl().sync()
+    return {"ok": True, "site_id": sid, "ttl_path": str(path)}
+
+
 @router.post("/import")
 def import_model(body: ImportBody, user: dict = Depends(require_roles("integrator"))) -> dict:
```

Ruff (0.15.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)